### ISSUE TITLE
Error: "Cannot set property 'min(datum)' of undefined" on empty db 

### ISSUE THREAD

**Issue by nielsnl68** [ujjwalguptaofficial/JsStore#220]
I'm getting this error on an empty db.

Is that as intended and should i use the "ignore" value here as well?

``` javascript
        let lastDate = await jsstoreCon.select({
            from: "metingen",
            where: {
                meter: ref * 1,
                //         datum: { ">": "20210505" }
            },
            aggregate: {
                min: 'datum'
            }
        });
```

**Comment by ujjwalguptaofficial** [comment]
Its a bug

**Comment by ujjwalguptaofficial** [comment]
fixed in v - 4.2.2.
